<!DOCTYPE html>
<html lang="en-US">
    <head>

        <meta charset="UTF-8">
<!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-2C44LTKBE1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-2C44LTKBE1');
</script>
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
        <script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
        
            <title>Biophysics - Nucleic Acids</title>
    
        <!-- Meta tags -->
        
        <meta name="keywords" content="nucleic acid, A-form, B-form, Z-form">
        <meta name="author" content="John Della Rosa" >
        <meta name="description" content="This chapter covers thermodynamics and chemistry of nucleic acids. Molecular structure and higher order structures are covered.">
        
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
    
        <link rel="apple-touch-icon" sizes="180x180" href="https://johndellarosa.github.io/apple-touch-icon.png">
        <link rel="icon" type="image/png" sizes="32x32" href="https://johndellarosa.github.io/favicon-32x32.png">
        <link rel="icon" type="image/png" sizes="16x16" href="https://johndellarosa.github.io/favicon-16x16.png">
        <link rel="manifest" href="https://johndellarosa.github.io/site.webmanifest">
        <link rel="canonical" href="https://johndellarosa.github.io/projects/biophysics-book/nucleic-acids"/>    
        <link rel="stylesheet" href="https://johndellarosa.github.io/style.css"> 
        <!-- </script> -->
        <!-- <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.4/jquery.min.js"></script>
      

        <script src="../../math.js" type="text/javascript"></script> -->
<!-- JSON-LD markup generated by Google Structured Data Markup Helper. -->
<script type="application/ld+json">
    [
      {
        "@context": "http://schema.org",
        "@type": "Article",
        "name": "Nucleic Acids Biophysics",
        "author": {
          "@type": "Person",
          "name": "John Della Rosa"
        },
        "articleSection": "Nucleic Acid Structure",
        "articleBody": "Primary Structure</H3>\n                <H4>Nitrogeneous Bases</H4>\n                <H5>Purines</H5>\n\n                There are two purines that are in DNA and RNA:\n                <DIV class=\"grid-container\" style=\"grid-template-columns: 50% 50%; justify-content:space-around;\">\n                    <DIV class=\"grid-item\">\n                        <H2>Adenine</H2>\n                        \n                        <BR/>\n                        <P style=\"width: 70%; text-align: center; margin: auto;\"><A href=\"https://commons.wikimedia.org/wiki/File:Adenine.svg#/media/File:Adenine.svg\"><IMG src=\"https://upload.wikimedia.org/wikipedia/commons/d/db/Adenine.svg\" alt=\"Adenine.svg\" width=\"70%\"/></A><BR/>By <A href=\"//commons.wikimedia.org/w/index.php?title=User:Pepemonbu&amp;amp;action=edit&amp;amp;redlink=1\" class=\"new\" title=\"User:Pepemonbu (page does not exist)\">Pepemonbu</A> - Made with BkChem and Inkscape., <A href=\"http://creativecommons.org/licenses/by-sa/3.0/\" title=\"Creative Commons Attribution-Share Alike 3.0\">CC BY-SA 3.0</A>, <A href=\"https://commons.wikimedia.org/w/index.php?curid=5581319\">Link</A></P>\n                </DIV>\n                    <DIV class=\"grid-item\">\n                        <H2>Guanine</H2>\n                        \n                        <BR/>\n                        <P style=\"width: 70%; text-align: center; margin: auto;\"><A href=\"https://commons.wikimedia.org/wiki/File:Guanin.svg#/media/File:Guanin.svg\"><IMG src=\"https://upload.wikimedia.org/wikipedia/commons/3/3f/Guanin.svg\" alt=\"Guanin.svg\" width=\"70%\"/></A><BR/>By <A href=\"//commons.wikimedia.org/wiki/User:NEUROtiker\" title=\"User:NEUROtiker\">NEUROtiker</A> - <SPAN class=\"int-own-work\" lang=\"en\">Own work</SPAN>, Public Domain, <A href=\"https://commons.wikimedia.org/w/index.php?curid=2212115\">Link</A></P>\n                    </DIV>\n                </DIV>\n                <H5>Pyrimidines</H5>\n\n                Thymine only appears in DNA. In RNA, it is replaced by the similar Uracil. \n\n\n                <DIV class=\"grid-container\" style=\"grid-template-columns: 50% 50%; justify-content:space-around;\">\n                    <DIV class=\"grid-item\">\n                        <H2>Thymine</H2>\n                        \n                        <BR/>\n                        <P style=\"width: 70%; text-align: center; margin: auto;\"><A href=\"https://commons.wikimedia.org/wiki/File:Thymin.svg#/media/File:Thymin.svg\"><IMG src=\"https://upload.wikimedia.org/wikipedia/commons/1/15/Thymin.svg\" alt=\"Thymin.svg\" width=\"70%\"/></A><BR/>By <A href=\"//commons.wikimedia.org/wiki/User:NEUROtiker\" title=\"User:NEUROtiker\">NEUROtiker</A> - <SPAN class=\"int-own-work\" lang=\"en\">Own work</SPAN>, Public Domain, <A href=\"https://commons.wikimedia.org/w/index.php?curid=2231106\">Link</A></P>\n                    </DIV>\n                    <DIV class=\"grid-item\">\n                        <H2>Cytosine</H2>\n                        \n                        <BR/>\n                        <P style=\"width: 70%; text-align: center; margin: auto;\"><A href=\"https://commons.wikimedia.org/wiki/File:Cytosine_structure_2022.svg#/media/File:Cytosine_structure_2022.svg\"><IMG src=\"https://upload.wikimedia.org/wikipedia/commons/1/18/Cytosine_structure_2022.svg\" alt=\"Cytosine structure 2022.svg\" width=\"60%\"/></A><BR/>By <A href=\"//commons.wikimedia.org/w/index.php?title=User:Wedgetail494&amp;amp;action=edit&amp;amp;redlink=1\" class=\"new\" title=\"User:Wedgetail494 (page does not exist)\">Wedgetail494</A> - <SPAN class=\"int-own-work\" lang=\"en\">Own work</SPAN> Public Domain, <A href=\"https://commons.wikimedia.org/w/index.php?curid=121538307\">Link</A></P>\n                    </DIV>\n                    <DIV class=\"grid-item\">\n                        <H2>Uracil</H2>\n                        \n                        <BR/>\n                        <P style=\"width: 70%; text-align: center; margin: auto;\"><A href=\"https://commons.wikimedia.org/wiki/File:Uracil.svg#/media/File:Uracil.svg\"><IMG src=\"https://upload.wikimedia.org/wikipedia/commons/b/b4/Uracil.svg\" alt=\"Structural formula of uracil\" width=\"60%\"/></A><BR/>By <A href=\"//commons.wikimedia.org/wiki/User:NEUROtiker\" title=\"User:NEUROtiker\">NEUROtiker</A> - <SPAN class=\"int-own-work\" lang=\"en\">Own work</SPAN>, Public Domain, <A href=\"https://commons.wikimedia.org/w/index.php?curid=2230876\">Link</A></P>\n                    </DIV>\n\n                </DIV>\n                \n                <H4>Sugar</H4>\n                <H5>RNA vs DNA</H5>\n                Besides Uracil vs Thymine, there is another chemical difference between DNA and RNA. \n                RNA has a hydroxyl group on the 2&#39; carbon while there is only a hydrogen attached to the 2&#39; carbon in DNA. \n\n                <BR/>\n                The absence of the 2&#39; hydroxyl group in DNA&#39;s deoxyribose sugar contributes to its stability. The hydroxyl group in RNAâ€™s ribose makes RNA more susceptible to hydrolysis, especially under alkaline conditions, limiting its stability relative to DNA. It can nucleophilically attack the phosphorous in the phosphodiester linkage, catalyzing the breakage of the link between the two nucleotides. . This difference in stability is critical as DNA needs to store genetic information reliably, while RNA acts primarily in temporary roles like protein synthesis.\n\n                \n                <H3>Secondary Structure</H3>\n\n                <H4>Base Pairing</H4>\n                <H5>Hydrogen Bonding</H5>\n                Base-pairing involves hydrogen bonding between two bases on different helices. \n                This is not the main driver of the formation of double helices, but it does have the following advantages:\n                <OL>\n                    <LI>Allows for bases to come together without enthalpic penalty from not being able to hydrogen bond with water</LI>\n                    <LI>Allows for detection of bases and specificity of base-pairing</LI>\n                </OL>\n                <H5>Canonical Base Pairs</H5>\n                Also known as Watson-Crick base-pairs, these are the normal nucleic acid pairings in nucleic acids.\n                <UL>\n                    <LI>DNA:\n                        <UL>\n                            <LI>Adenine-Thymine (A-T): Forms two hydrogen bonds.</LI>\n                            <LI>Cytosine-Guanine (C-G): Forms three hydrogen bonds.</LI>\n                        </UL>\n                    </LI>\n                    <LI>RNA:\n                        <UL>\n                            <LI>Adenine-Uracil (A-U): Forms two hydrogen bonds.</LI>\n                            <LI>Cytosine-Guanine (C-G): Forms three hydrogen bonds.</LI>\n                        </UL>\n                    </LI>\n\n                </UL>\n\n                \n\n                <H5>Chargaff&#39;s Rules</H5>\n                Because A is always paired with T, the amount of A must be equal to the amount of T. A similar relationship holds between amount of G and C. \n                This is known as Chargaff&#39;s rules, and it holds very well in DNA, where there is very little mismatching.\n                <FIGURE>\n                <IMG src=\"chargaffs-rules.png\" alt=\"Chargaff&#39;s Rules\" style=\"margin:auto; display: block;  width:clamp(300px,50vw, 800px)\"/>\n                <FIGCAPTION>DNA Composition Showing Chargaff&#39;s Rules</FIGCAPTION>\n            </FIGURE>\n                \n            <BR/>\n                <BUTTON type=\"button\" class=\"collapsible\">Python Code</BUTTON>\n                \n                \n                <H5>Hoogsteen Base-Pairing</H5>\n\n                Hoogesteen base-pairing is an alternative hydrogen bonding pattern.\n\n                <H5>Wobble Base-Pairing</H5>\n                In tRNA, the final base in the codon is less restricted than traditional Watson-Crick base pairing. \n                <H5>Base Tautomerism</H5>\n                It is possible for the bases to undergo tautomerization. \n                Keto groups can convert to enols. Amines can convert to imines. Both of these changes reverse the role of that group being a hydrogen donor versus acceptor. \n                This changes the hydrogen bonding pattern, promoting base mismatching and thus mutations.\n                \n                <H4>Double Helix</H4>\n                <H5>Base Stacking</H5>\n                Much of the free energetic drive for double helix formation comes from base stacking. \n                There is the enthalpic contribution from pi orbital overlap with the neighboring bases. \n                There is also an entropic contribution from water not having to solvate the bases.\n                <H6>Example: Ethidium Bromide</H6>\n                Ethidium bromide is an <B>intercalating agent</B>, which means that it can bind between stacked bases. \n                Ethidium bromide is used for detection and visualization in gel electrophoresis. \n                \n                <H5>Antiparallel</H5>\n                <H5>Major and Minor Grooves</H5>\n                As the strands twist around each other, the spatial alignment of the bases results in the formation of two distinct grooves.\n\n                The major groove is wider in B form DNA, while the minor groove is roughly half as wide; however, the minor groove is deeper.\n                Because of the major groove&#39;s wideness, it provides more accessible space for proteins to interact because more atoms of the base pairs are exposed.\n\n\n\n\n                <H4>A Form, B Form, Z Form</H4>\n                <H5>A-form</H5>\n                <UL>\n                    <LI>Commonly seen in RNA double helices and DNA-RNA helices</LI>\n                    <LI>Right handed</LI>\n                    <LI>11 base pairs per turn</LI>\n                    <LI>Helix is shorter and wider than B-form</LI>\n                    <LI>Deep and narrow major groove</LI>\n                </UL>\n                <H5>B-form</H5>\n                <UL>\n                    <LI>Common conformation for DNA</LI>\n                    <LI>Right-handed</LI>\n                    <LI>10 base pairs per turn</LI>\n                    <LI>Wide, shallow major groove</LI>\n                </UL>\n                <H5>Z-form</H5>\n                <UL>\n                    <LI>Left-handed</LI>\n                    <LI>12 base pairs per turn</LI>\n                    <LI>Less distinction between major and minor grooves</LI>\n                </UL>\n                <H4>RNA motifs</H4>\n                Since RNA commonly exists in a single-stranded state, unlimke DNA, more variety in structure is observed.\n\n\n                <H5>Stem-loops</H5>\n                    Single strand RNA base pairs with itself, forming a stem. Unpaired bases between the two segments form the loop. Also known as a hairpin loop.\n\n                <H5>Bulges</H5>\n                Bulges are short unpaired regions in between paired regions.\n\n                <H3>Higher Structures</H3>\n                <H4>Supercoiling</H4>\n                While there is natural twisting due to the local twisting along the helix, there can also be a global twisting. \n\n                <H5>Twist</H5>\n                This refers to the number of helical turns in the DNA double helix. Imagine a ladder that represents the DNA. If you were to count the number of rungs (base pairs) and then determine how many times one side of the ladder spirals around the other, you would be calculating the twist. In canonical B-DNA, there are approximately 10.5 base pairs per turn (or twist). Changes in the twist would mean either overwinding or underwinding from this standard configuration.\n                <H5>Writhe</H5>\n                This describes the coiling of the DNA helix axis in space, almost like the loops in a roller coaster track. It signifies how many times the double helix crosses over itself. Positive writhe indicates right-handed supercoiling, while negative writhe indicates left-handed supercoiling.\n                \n                <H5>Linking Number</H5>\n                Linking number is the number of times one DNA strand wraps around the other. \n\n                It is a topological property, meaning it remains constant unless DNA is broken. Lk can be partitioned into the sum of twist and writhe: <MJX-CONTAINER class=\"MathJax CtxtMenu_Attached_0\" jax=\"CHTML\" tabindex=\"0\" ctxtmenu_counter=\"0\" style=\"font-size: 117.4%; position: relative;\"><MJX-MATH class=\"MJX-TEX\" aria-hidden=\"true\"><MJX-MI class=\"mjx-i\"><MJX-C class=\"mjx-c1D43F TEX-I\"></MJX-C></MJX-MI><MJX-MI class=\"mjx-i\"><MJX-C class=\"mjx-c1D458 TEX-I\"></MJX-C></MJX-MI><MJX-MO class=\"mjx-n\" space=\"4\"><MJX-C class=\"mjx-c3D\"></MJX-C></MJX-MO><MJX-MI class=\"mjx-i\" space=\"4\"><MJX-C class=\"mjx-c1D447 TEX-I\"></MJX-C></MJX-MI><MJX-MI class=\"mjx-i\"><MJX-C class=\"mjx-c1D464 TEX-I\"></MJX-C></MJX-MI><MJX-MO class=\"mjx-n\" space=\"3\"><MJX-C class=\"mjx-c2B\"></MJX-C></MJX-MO><MJX-MI class=\"mjx-i\" space=\"3\"><MJX-C class=\"mjx-c1D44A TEX-I\"></MJX-C></MJX-MI><MJX-MI class=\"mjx-i\"><MJX-C class=\"mjx-c1D45F TEX-I\"></MJX-C></MJX-MI></MJX-MATH><MJX-ASSISTIVE-MML unselectable=\"on\" display=\"inline\"><math xmlns=\"http://www.w3.org/1998/Math/MathML\"><mi>L</mi><mi>k</mi><mo>=</mo><mi>T</mi><mi>w</mi><mo>+</mo><mi>W</mi><mi>r"
      },
      {
        "@context": "http://schema.org",
        "@type": "Article",
        "articleSection": "Physical Properties of Nucleic Acids",
        "articleBody": "Denaturation</H3>\n                <FIGURE>\n                <IMG src=\"nucleic-acid-melting-curve.png\" alt=\"DNA Melting Curve\" style=\"margin:auto; display: block;  width:clamp(300px,50vw, 800px)\"/>\n                <FIGCAPTION>DNA Melting Curve Example</FIGCAPTION>\n            </FIGURE>\n\n                <H4>GC Content</H4>\n                Because the GC pair has 3 hydrogen bonds while AT has 2, DNA that has a higher percentage of G&#39;s and C&#39;s in it will be harder to separate. \n                This is an evolutionary trick that is used by organisms that live in high temperature environments. \n                \n                \n                <H3>UV Absorption</H3>\n                When UV light interacts with molecules, electrons can be excited from their ground state to an excited state. For nucleic acids, the primary absorbing components are the nitrogenous bases. They strongly absorb UV light around 260 nm, making this wavelength ideal for nucleic acid quantification.\n\n                <H4>Hyperchromic Effect</H4>\n                Upon denaturation (e.g., heating), the UV absorbance of nucleic acids increases, a phenomenon called the hyperchromic effect. This is because the stacked bases in the double-helical structure become unstacked upon denaturation, leading to <A href=\"https://pubs.acs.org/doi/10.1021/jp403369k\">increased delocalization</A> and hence higher absorption. Monitoring the hyperchromic effect can help determine the melting temperature <MJX-CONTAINER class=\"MathJax CtxtMenu_Attached_0\" jax=\"CHTML\" tabindex=\"0\" ctxtmenu_counter=\"1\" style=\"font-size: 117.4%; position: relative;\"><MJX-MATH class=\"MJX-TEX\" aria-hidden=\"true\"><MJX-MO class=\"mjx-n\"><MJX-C class=\"mjx-c28\"></MJX-C></MJX-MO><MJX-MSUB><MJX-MI class=\"mjx-i\"><MJX-C class=\"mjx-c1D447 TEX-I\"></MJX-C></MJX-MI><MJX-SCRIPT style=\"vertical-align: -0.15em; margin-left: -0.12em;\"><MJX-MI class=\"mjx-i\" size=\"s\"><MJX-C class=\"mjx-c1D45A TEX-I\"></MJX-C></MJX-MI></MJX-SCRIPT></MJX-MSUB><MJX-MO class=\"mjx-n\"><MJX-C class=\"mjx-c29\"></MJX-C></MJX-MO></MJX-MATH><MJX-ASSISTIVE-MML unselectable=\"on\" display=\"inline\"><math xmlns=\"http://www.w3.org/1998/Math/MathML\"><mo stretchy=\"false\">(</mo><msub><mi>T</mi><mi>m</mi></msub><mo stretchy=\"false\">)</mo></math></MJX-ASSISTIVE-MML></MJX-CONTAINER> of nucleic acids, which is a measure of their stability.\n                <H4>Uses</H4>\n                <H5>Quantification of Nucleic Acids</H5>\n                Since the absorption of UV light is proportional to the concentration of nucleic acids, spectrophotometers are used to determine the concentration of DNA or RNA in a sample by measuring its absorbance at 260 nm.\n\n                <H5>Purity Assessment of Nucleic Acids</H5>\n                The ratio of absorbance at 260 nm to 280 nm (A260/A280 ratio) provides information about the purity of a nucleic acid sample. Pure DNA typically has an A260/A280 ratio of ~1.8, while pure RNA has a ratio of ~2.0. Deviations from these values can indicate contamination, often from proteins."
      },
      {
        "@context": "http://schema.org",
        "@type": "Article",
        "articleSection": "Nucleic Acid Practice Questions",
        "articleBody": "Describe the structural differences and similarities between DNA and RNA.</LI>\n                    <LI>What is the role of the phosphate group in the structure and function of nucleic acids?</LI>\n                    <LI>Explain the significance of base-pairing in the structure and function of nucleic acids.</LI>\n                    <LI>How do hydrogen bonds facilitate base-pairing in nucleic acids?</LI>\n                    <LI>What would be the consequences of non-canonical base-pairing for the structure and stability of nucleic acids?</LI>\n                    <LI>Describe the structural differences between A-form, B-form, and Z-form of DNA.</LI>\n                    <LI>Define GC content and explain its significance in nucleic acid stability.</LI>\n                    <LI>How does GC content affect the melting temperature of a DNA molecule?</LI>\n                    <LI>Explain the concept of melting temperature in the context of nucleic acid structure.</LI>\n                    <LI>How do the number and type of base pairs in a nucleic acid strand influence its melting temperature?</LI>\n                    <LI>Distinguish between purines and pyrimidines in terms of structure and examples.</LI>\n                    <LI>Why do purines pair with pyrimidines in the structure of nucleic acids?</LI>\n                    <LI>How does the melting temperature correlate with the thermophilic nature of some organisms?</LI>\n                    <LI>Contrast the major and minor groove."
      }
    ]
    </script>
    </head>
    <body>
        <div class="navbar">
            <b style="margin-right:10px">John Della Rosa</b>| 
            <a href="https://johndellarosa.github.io/">Home</a>|
            <a href="https://johndellarosa.github.io/resume">Resume</a>|
            <a href="https://johndellarosa.github.io/biography">About</a>|
            <a href="https://johndellarosa.github.io/projects">Projects</a>|
            <a href="https://johndellarosa.github.io/miscellaneous">Misc</a>|
            <a href="https://www.linkedin.com/in/johndellarosa/" target="_blank">Linkedin</a>|
            <a href="https://github.com/johndellarosa" target="_blank">Github</a>|
            <a href="https://play.google.com/store/apps/developer?id=JohnDellaRosa" target="_blank">Google Play</a>|
            <a href="https://apps.apple.com/us/developer/john-della-rosa/id1684177615" target="_blank">Apple Store</a>
        
        </div>
        <h2><a href= "table-of-contents.html">Biophysical Chemistry Textbook (Work in Progress)</a></h2>

<h3>by John Della Rosa</h3>
        <div id="text-contents" style="width:90%; margin:auto">
            <div id="Nucleic-Acids">

                <h1>Nucleic Acids Biophysics</h1>
                <h2>Nucleic Acid Structure</h2>
                <h3>Primary Structure</h3>
                <h4>Nitrogeneous Bases</h4>
                <h5>Purines</h5>

                There are two purines that are in DNA and RNA:
                <div class="grid-container" style="grid-template-columns: 50% 50%; justify-content:space-around;">
                    <div class="grid-item">
                        <h2>Adenine</h2>
                        
                        <br>
                        <p style="width: 70%; text-align: center; margin: auto;"><a href="https://commons.wikimedia.org/wiki/File:Adenine.svg#/media/File:Adenine.svg"><img src="https://upload.wikimedia.org/wikipedia/commons/d/db/Adenine.svg" alt="Adenine.svg" width="70%"></a><br>By <a href="//commons.wikimedia.org/w/index.php?title=User:Pepemonbu&amp;amp;action=edit&amp;amp;redlink=1" class="new" title="User:Pepemonbu (page does not exist)">Pepemonbu</a> - Made with BkChem and Inkscape., <a href="http://creativecommons.org/licenses/by-sa/3.0/" title="Creative Commons Attribution-Share Alike 3.0">CC BY-SA 3.0</a>, <a href="https://commons.wikimedia.org/w/index.php?curid=5581319">Link</a></p>
                </div>
                    <div class="grid-item">
                        <h2>Guanine</h2>
                        
                        <br>
                        <p style="width: 70%; text-align: center; margin: auto;"><a href="https://commons.wikimedia.org/wiki/File:Guanin.svg#/media/File:Guanin.svg"><img src="https://upload.wikimedia.org/wikipedia/commons/3/3f/Guanin.svg" alt="Guanin.svg" width="70%"></a><br>By <a href="//commons.wikimedia.org/wiki/User:NEUROtiker" title="User:NEUROtiker">NEUROtiker</a> - <span class="int-own-work" lang="en">Own work</span>, Public Domain, <a href="https://commons.wikimedia.org/w/index.php?curid=2212115">Link</a></p>
                    </div>
                </div>
                <h5>Pyrimidines</h5>

                Thymine only appears in DNA. In RNA, it is replaced by the similar Uracil. 


                <div class="grid-container" style="grid-template-columns: 50% 50%; justify-content:space-around;">
                    <div class="grid-item">
                        <h2>Thymine</h2>
                        
                        <br>
                        <p style="width: 70%; text-align: center; margin: auto;"><a href="https://commons.wikimedia.org/wiki/File:Thymin.svg#/media/File:Thymin.svg"><img src="https://upload.wikimedia.org/wikipedia/commons/1/15/Thymin.svg" alt="Thymin.svg" width="70%"></a><br>By <a href="//commons.wikimedia.org/wiki/User:NEUROtiker" title="User:NEUROtiker">NEUROtiker</a> - <span class="int-own-work" lang="en">Own work</span>, Public Domain, <a href="https://commons.wikimedia.org/w/index.php?curid=2231106">Link</a></p>
                    </div>
                    <div class="grid-item">
                        <h2>Cytosine</h2>
                        
                        <br>
                        <p style="width: 70%; text-align: center; margin: auto;"><a href="https://commons.wikimedia.org/wiki/File:Cytosine_structure_2022.svg#/media/File:Cytosine_structure_2022.svg"><img src="https://upload.wikimedia.org/wikipedia/commons/1/18/Cytosine_structure_2022.svg" alt="Cytosine structure 2022.svg" width="60%"></a><br>By <a href="//commons.wikimedia.org/w/index.php?title=User:Wedgetail494&amp;amp;action=edit&amp;amp;redlink=1" class="new" title="User:Wedgetail494 (page does not exist)">Wedgetail494</a> - <span class="int-own-work" lang="en">Own work</span> Public Domain, <a href="https://commons.wikimedia.org/w/index.php?curid=121538307">Link</a></p>
                    </div>
                    <div class="grid-item">
                        <h2>Uracil</h2>
                        
                        <br>
                        <p style="width: 70%; text-align: center; margin: auto;"><a href="https://commons.wikimedia.org/wiki/File:Uracil.svg#/media/File:Uracil.svg"><img src="https://upload.wikimedia.org/wikipedia/commons/b/b4/Uracil.svg" alt="Structural formula of uracil" width="60%"></a><br>By <a href="//commons.wikimedia.org/wiki/User:NEUROtiker" title="User:NEUROtiker">NEUROtiker</a> - <span class="int-own-work" lang="en">Own work</span>, Public Domain, <a href="https://commons.wikimedia.org/w/index.php?curid=2230876">Link</a></p>
                    </div>

                </div>
                
                <h4>Sugar</h4>
                <h5>RNA vs DNA</h5>
                Besides Uracil vs Thymine, there is another chemical difference between DNA and RNA. 
                RNA has a hydroxyl group on the 2' carbon while there is only a hydrogen attached to the 2' carbon in DNA. 

                <br>
                The absence of the 2' hydroxyl group in DNA's deoxyribose sugar contributes to its stability. The hydroxyl group in RNAâ€™s ribose makes RNA more susceptible to hydrolysis, especially under alkaline conditions, limiting its stability relative to DNA. It can nucleophilically attack the phosphorous in the phosphodiester linkage, catalyzing the breakage of the link between the two nucleotides. . This difference in stability is critical as DNA needs to store genetic information reliably, while RNA acts primarily in temporary roles like protein synthesis.

                
                <h3>Secondary Structure</h3>

                <h4>Base Pairing</h4>
                <h5>Hydrogen Bonding</h5>
                Base-pairing involves hydrogen bonding between two bases on different helices. 
                This is not the main driver of the formation of double helices, but it does have the following advantages:
                <ol>
                    <li>Allows for bases to come together without enthalpic penalty from not being able to hydrogen bond with water</li>
                    <li>Allows for detection of bases and specificity of base-pairing</li>
                </ol>
                <h5>Canonical Base Pairs</h5>
                Also known as Watson-Crick base-pairs, these are the normal nucleic acid pairings in nucleic acids.
                <ul>
                    <li>DNA:
                        <ul>
                            <li>Adenine-Thymine (A-T): Forms two hydrogen bonds.</li>
                            <li>Cytosine-Guanine (C-G): Forms three hydrogen bonds.</li>
                        </ul>
                    </li>
                    <li>RNA:
                        <ul>
                            <li>Adenine-Uracil (A-U): Forms two hydrogen bonds.</li>
                            <li>Cytosine-Guanine (C-G): Forms three hydrogen bonds.</li>
                        </ul>
                    </li>

                </ul>

                

                <h5>Chargaff's Rules</h5>
                Because A is always paired with T, the amount of A must be equal to the amount of T. A similar relationship holds between amount of G and C. 
                This is known as Chargaff's rules, and it holds very well in DNA, where there is very little mismatching.
                <figure>
                <img src="chargaffs-rules.png" alt="Chargaff's Rules" style="margin:auto; display: block;  width:clamp(300px,50vw, 800px)"/>
                <figcaption>DNA Composition Showing Chargaff's Rules</figcaption>
            </figure>
                
            <br>
                <button type="button" class="collapsible">Python Code</button>
                <div class="content">
                <code style="margin: auto; width: 50%;"><pre>
import matplotlib.pyplot as plt
import numpy as np

def plot_chargaffs_rules():
    bases = ['A', 'T', 'C', 'G']
    sns.set_theme(style="whitegrid")
    # Fictional relative amounts
    amounts = [20, 20, 30, 30]  # A:T and C:G have the same amounts
    
    fig, ax = plt.subplots(figsize=(8, 6))
    
    bars = sns.barplot(x=bases, y=amounts,
                       palette=['tab:red','tab:blue','tab:green','tab:orange'],
                       ax=ax,edgecolor='k')
    
    # Highlighting the equal amounts


    for p in ax.patches:
        h, w, x = p.get_height(), p.get_width(), p.get_x()
        xy = (x + w / 2., h / 2)
        text = f'{h:0.0f}%'
        ax.annotate(text=text, xy=xy, ha='center', va='center',color='white',size=18)

    ax.set_ylim(0, max(amounts) + 5)
    ax.set_xlabel('Nucleotide Bases')
    ax.set_ylabel('Relative Amount (%)')
    ax.set_title("Illustration of Chargaff's Rules")
    #ax.grid(axis='y', linestyle='--', linewidth=0.7, alpha=0.7)

    plt.tight_layout()
    plt.show()

plot_chargaffs_rules()

                    </pre>
                </code></div>
                
                <h5>Hoogsteen Base-Pairing</h5>

                Hoogesteen base-pairing is an alternative hydrogen bonding pattern.

                <h5>Wobble Base-Pairing</h5>
                In tRNA, the final base in the codon is less restricted than traditional Watson-Crick base pairing. 
                <h5>Base Tautomerism</h5>
                It is possible for the bases to undergo tautomerization. 
                Keto groups can convert to enols. Amines can convert to imines. Both of these changes reverse the role of that group being a hydrogen donor versus acceptor. 
                This changes the hydrogen bonding pattern, promoting base mismatching and thus mutations.
                
                <h4>Double Helix</h4>
                <h5>Base Stacking</h5>
                Much of the free energetic drive for double helix formation comes from base stacking. 
                There is the enthalpic contribution from pi orbital overlap with the neighboring bases. 
                There is also an entropic contribution from water not having to solvate the bases.
                <h6>Example: Ethidium Bromide</h6>
                Ethidium bromide is an <b>intercalating agent</b>, which means that it can bind between stacked bases. 
                Ethidium bromide is used for detection and visualization in gel electrophoresis. 
                
                <h5>Antiparallel</h5>
                <h5>Major and Minor Grooves</h5>
                As the strands twist around each other, the spatial alignment of the bases results in the formation of two distinct grooves.

                The major groove is wider in B form DNA, while the minor groove is roughly half as wide; however, the minor groove is deeper.
                Because of the major groove's wideness, it provides more accessible space for proteins to interact because more atoms of the base pairs are exposed.




                <h4>A Form, B Form, Z Form</h4>
                <h5>A-form</h5>
                <ul>
                    <li>Commonly seen in RNA double helices and DNA-RNA helices</li>
                    <li>Right handed</li>
                    <li>11 base pairs per turn</li>
                    <li>Helix is shorter and wider than B-form</li>
                    <li>Deep and narrow major groove</li>
                </ul>
                <h5>B-form</h5>
                <ul>
                    <li>Common conformation for DNA</li>
                    <li>Right-handed</li>
                    <li>10 base pairs per turn</li>
                    <li>Wide, shallow major groove</li>
                </ul>
                <h5>Z-form</h5>
                <ul>
                    <li>Left-handed</li>
                    <li>12 base pairs per turn</li>
                    <li>Less distinction between major and minor grooves</li>
                </ul>
                <h4>RNA motifs</h4>
                Since RNA commonly exists in a single-stranded state, unlimke DNA, more variety in structure is observed.


                <h5>Stem-loops</h5>
                    Single strand RNA base pairs with itself, forming a stem. Unpaired bases between the two segments form the loop. Also known as a hairpin loop.

                <h5>Bulges</h5>
                Bulges are short unpaired regions in between paired regions.

                <h3>Higher Structures</h3>
                <h4>Supercoiling</h4>
                While there is natural twisting due to the local twisting along the helix, there can also be a global twisting. 

                <h5>Twist</h5>
                This refers to the number of helical turns in the DNA double helix. Imagine a ladder that represents the DNA. If you were to count the number of rungs (base pairs) and then determine how many times one side of the ladder spirals around the other, you would be calculating the twist. In canonical B-DNA, there are approximately 10.5 base pairs per turn (or twist). Changes in the twist would mean either overwinding or underwinding from this standard configuration.
                <h5>Writhe</h5>
                This describes the coiling of the DNA helix axis in space, almost like the loops in a roller coaster track. It signifies how many times the double helix crosses over itself. Positive writhe indicates right-handed supercoiling, while negative writhe indicates left-handed supercoiling.
                
                <h5>Linking Number</h5>
                Linking number is the number of times one DNA strand wraps around the other. 

                It is a topological property, meaning it remains constant unless DNA is broken. Lk can be partitioned into the sum of twist and writhe: \(Lk=Tw+Wr\)

                <h3>Epigenetics</h3>
                Epigenetics explores the mechanisms by which gene expression is regulated without alterations to the underlying DNA sequence. This layer of genetic control is crucial for many biological processes, including development, differentiation, and response to environmental stimuli. 

                <h4>Common Types</h4>
                <h5>DNA Methylation</h5>
                Methylation of cytosine bases, particularly at CpG dinucleotides, is a common epigenetic mark. DNA methylation generally represses gene expression by obstructing the binding of transcription factors or recruiting methyl-binding proteins that further recruit repressive chromatin modifiers.

                <h5>Histone Modification</h5>
                Histones, the protein spools around which DNA is wrapped, can be modified in various ways, including acetylation, methylation, phosphorylation, and ubiquitination. These modifications alter the structure of chromatin (the complex of DNA and histones) and, consequently, the accessibility of DNA to the transcriptional machinery.
                <h2>Physical Properties of Nucleic Acids</h2>
                <h3>Denaturation</h3>
                <figure>
                <img src="nucleic-acid-melting-curve.png" alt="DNA Melting Curve" style="margin:auto; display: block;  width:clamp(300px,50vw, 800px)"/>
                <figcaption>DNA Melting Curve Example</figcaption>
            </figure>

                <h4>GC Content</h4>
                Because the GC pair has 3 hydrogen bonds while AT has 2, DNA that has a higher percentage of G's and C's in it will be harder to separate. 
                This is an evolutionary trick that is used by organisms that live in high temperature environments. 
                
                
                <h3>UV Absorption</h3>
                When UV light interacts with molecules, electrons can be excited from their ground state to an excited state. For nucleic acids, the primary absorbing components are the nitrogenous bases. They strongly absorb UV light around 260 nm, making this wavelength ideal for nucleic acid quantification.

                <h4>Hyperchromic Effect</h4>
                Upon denaturation (e.g., heating), the UV absorbance of nucleic acids increases, a phenomenon called the hyperchromic effect. This is because the stacked bases in the double-helical structure become unstacked upon denaturation, leading to <a href="https://pubs.acs.org/doi/10.1021/jp403369k">increased delocalization</a> and hence higher absorption. Monitoring the hyperchromic effect can help determine the melting temperature \((T_m)\) of nucleic acids, which is a measure of their stability.
                <h4>Uses</h4>
                <h5>Quantification of Nucleic Acids</h5>
                Since the absorption of UV light is proportional to the concentration of nucleic acids, spectrophotometers are used to determine the concentration of DNA or RNA in a sample by measuring its absorbance at 260 nm.

                <h5>Purity Assessment of Nucleic Acids</h5>
                The ratio of absorbance at 260 nm to 280 nm (A260/A280 ratio) provides information about the purity of a nucleic acid sample. Pure DNA typically has an A260/A280 ratio of ~1.8, while pure RNA has a ratio of ~2.0. Deviations from these values can indicate contamination, often from proteins.

                <h2>Chemistry of Nucleic Acids</h2>
                <h3>Energy Storage</h3>
                <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7032660/">Why Nature Really Chose Phosphate</a>

            </div>

            <div id="Exercises">
                <h2>Nucleic Acid Practice Questions</h2>
                <ol>
                    <li>Describe the structural differences and similarities between DNA and RNA.</li>
                    <li>What is the role of the phosphate group in the structure and function of nucleic acids?</li>
                    <li>Explain the significance of base-pairing in the structure and function of nucleic acids.</li>
                    <li>How do hydrogen bonds facilitate base-pairing in nucleic acids?</li>
                    <li>What would be the consequences of non-canonical base-pairing for the structure and stability of nucleic acids?</li>
                    <li>Describe the structural differences between A-form, B-form, and Z-form of DNA.</li>
                    <li>Define GC content and explain its significance in nucleic acid stability.</li>
                    <li>How does GC content affect the melting temperature of a DNA molecule?</li>
                    <li>Explain the concept of melting temperature in the context of nucleic acid structure.</li>
                    <li>How do the number and type of base pairs in a nucleic acid strand influence its melting temperature?</li>
                    <li>Distinguish between purines and pyrimidines in terms of structure and examples.</li>
                    <li>Why do purines pair with pyrimidines in the structure of nucleic acids?</li>
                    <li>How does the melting temperature correlate with the thermophilic nature of some organisms?</li>
                    <li>Contrast the major and minor groove.</li>








                </ol>


            </div>

        </div>
        <script>
            var coll = document.getElementsByClassName("collapsible");
            var i;
    
            for (i = 0; i < coll.length; i++) {
            coll[i].addEventListener("click", function() {
                this.classList.toggle("active");
                var content = this.nextElementSibling;
                if (content.style.display === "block") {
                content.style.display = "none";
                } else {
                content.style.display = "block";
                }
            });
            } 
            </script>
</body>
</html>