<!DOCTYPE html>
<html lang="en-US">
    <head>

        <meta charset="UTF-8">
<!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-2C44LTKBE1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-2C44LTKBE1');
</script>
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
        <script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
        
            <title>Protein Folding</title>
    
        <!-- Meta tags -->
        
        <meta name="keywords" content="Biophysics, protein folding, thermodynamics, alpha helix, beta sheet, amino acid">
        <meta name="author" content="John Della Rosa" >
        <meta name="description" content="This chapter covers the thermodynamics of protein folding. The primary, secondary, tertiary, and quartenary structures are introduced.">
        
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
    
        <link rel="apple-touch-icon" sizes="180x180" href="https://johndellarosa.github.io/apple-touch-icon.png">
        <link rel="icon" type="image/png" sizes="32x32" href="https://johndellarosa.github.io/favicon-32x32.png">
        <link rel="icon" type="image/png" sizes="16x16" href="https://johndellarosa.github.io/favicon-16x16.png">
        <link rel="manifest" href="https://johndellarosa.github.io/site.webmanifest">
            
        <link rel="stylesheet" href="https://johndellarosa.github.io/style.css"> 
        <!-- </script>
        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.4/jquery.min.js"></script>
      

        <script src="../../math.js" type="text/javascript"></script> -->

    </head>
    <body>
        <div class="navbar">
            <b style="margin-right:10px">John Della Rosa</b>| 
            <a href="https://johndellarosa.github.io/index.html">Home</a>|
            <a href="https://johndellarosa.github.io/resume.html">Resume</a>
            <!-- <a href="./index.html#education">Education</a>
            <a href="./index.html#experience">Experience</a>
            <a href="./index.html#skills">Skills</a> -->
            |
            <a href="https://johndellarosa.github.io/biography.html">About</a>|
            <a href="https://johndellarosa.github.io/projects.html">Projects</a>|
            <a href="https://johndellarosa.github.io/Miscellaneous.html">Misc</a>|
            <a href="https://www.linkedin.com/in/johndellarosa/" target="_blank">Linkedin</a>|
            <a href="https://github.com/johndellarosa" target="_blank">Github</a>|
            <a href="https://play.google.com/store/apps/developer?id=JohnDellaRosa" target="_blank">Google Play</a>|
            <a href="https://apps.apple.com/us/developer/john-della-rosa/id1684177615" target="_blank">Apple Store</a>
        
        </div>
        <h2><a href= "Table-of-Contents.html">Biophysical Chemistry Textbook (Work in Progress)</a></h2>

<h3>by John Della Rosa</h3>
        <div id="text-contents" style="width:90%; margin:auto">
            <div id="Protein-Folding">
                <h1> Protein Folding</h1>
                <h2>Introduction</h2>
                <h3>Prerequisites</h3>
                <ul>
                    <li>This chapter will not contain much math. However, an understanding of optimization will be useful.</li>
                    <li>Some basic thermodynamics knowledge will be assumed. You should understand the basics of:
                        <ul>
                            <li>What entropy and enthalpy are, at least at a conceptual level</li>
                            <li>The various types of intermolecular and intramolecular forces</li>
                            
                        </ul>
                    </li>
                    <li>Basic organic chemistry knowledge will be assumed. You should be familiar with the following:
                        <ul>
                            <li>Common functional group names and structures</li>
                            <li>Conjugation/Resonance</li>
                        </ul>
                    </li>
                </ul>


                <h2>Thermodynamics</h2>
                <h3>Enthalpy and Entropy</h3>
                The concepts of enthalpy and entropy will be focused on in this chapter. If you are uncomfortable with them, then you should read the earlier chapters on thermodynamics. 

                <br><br>

                What is the primary driver of protein folding? It is the <b>hydrophobic effect</b>. The hydrophobic effect is where the hydrophobic parts of the protein are burried together away from water and only the hydrophilic parts of the protein are exposed. 
                When the hydrophobic parts of the protein are burried, water molecules are freed from having to form an ordered shell. This allows for more hydrogen bonding, but even moreso, an increase in entropy for the water excluded. 
                

                <h2>Structures</h2>
                <h3>Primary Structure</h3>
                The primary structure of a protein is the sequence of amino acids, connected by amide linkages between the carboxylate group of the earlier one in the sequence and the amine group of the following one. 



                <h3>Secondary Structure</h3>
                The secondary structure involves hydrogen bonding between amide backbones of residues. There are two main conformations that the protein can fold itself into to do this. 
                <ul>
                    <li>Alpha helix</li>
                    <li>Beta sheet</li>
                </ul>
                There are a few other secondary structures that occur, but they are so rare that they are not worth mentioning. 
                A single protein can have many instances of both alpha helices and beta sheets in it. 


                <h4>Phi, Psi, and Omega</h4>
                When talking about conformations that the peptides are in, it becomes useful to consider 3 different bonds and the dihedral angles along them. 

                <ol>
                    <li>Omega: the peptide bond</li>
                    <li>Phi: the bond between the alpha carbon and the nitrogen</li>
                    <li>Psi: the bond between the alpha carbon and the carbonyl carbon</li>
                </ol>

                <p style="width: 70%; text-align: center; margin: auto;"><a href="https://commons.wikimedia.org/wiki/File:Protein_backbone_PhiPsiOmega_drawing.svg#/media/File:Protein_backbone_PhiPsiOmega_drawing.svg"><img src="https://upload.wikimedia.org/wikipedia/commons/9/97/Protein_backbone_PhiPsiOmega_drawing.svg" alt="Protein backbone PhiPsiOmega drawing.svg" height="988" width="520"></a><br>By <a href="//commons.wikimedia.org/wiki/User:Dcrjsr" title="User:Dcrjsr">Dcrjsr</a>, vectorised <a href="//commons.wikimedia.org/wiki/User:Adam_R%C4%99dzikowski" title="User:Adam Rędzikowski">Adam Rędzikowski</a> - <span class="int-own-work" lang="en">Own work</span>, <a href="https://creativecommons.org/licenses/by/3.0" title="Creative Commons Attribution 3.0">CC BY 3.0</a>, <a href="https://commons.wikimedia.org/w/index.php?curid=24585750">Link</a></p>

                <h4>Ramachandran Diagram</h4>

                <p style="width: 70%; text-align: center; margin: auto;"><a href="https://commons.wikimedia.org/wiki/File:Ramachandran_plot_original_outlines.jpg#/media/File:Ramachandran_plot_original_outlines.jpg"><img src="https://upload.wikimedia.org/wikipedia/commons/8/85/Ramachandran_plot_original_outlines.jpg" alt="Ramachandran plot original outlines.jpg" height="500" width="500"></a><br>By <a href="//commons.wikimedia.org/wiki/User:Dcrjsr" title="User:Dcrjsr">Dcrjsr</a> - <span class="int-own-work" lang="en">Own work</span>, <a href="https://creativecommons.org/licenses/by/3.0" title="Creative Commons Attribution 3.0">CC BY 3.0</a>, <a href="https://commons.wikimedia.org/w/index.php?curid=15230457">Link</a></p>
<br><br>
                In order for a secondary structure to form, the phi and psi angles of a series of consecutive residues must fall within certain regions. 
                For the right-handed alpha helix and beta sheets, phi is negative. The left-handed alpah helix is much rarer and is one of the few instances of having consecutive positive phi values. 
                <br><br>
                Omega is left out from the diagram as it is heavily restricted to either 0 or 180 degrees. This is in order to maintain conjugation of the pi system between the oxygen, carbon, and nitrogen of the amide. 
                Furthermore, omega has a strong preference for being at 180 degrees (trans, where the two alpha carbons are catty-corner) due to steric clashes between the two alpha carbons being on the same side of the C=N bond (cis). 
                Proline can be an exception when it is directly after the amide bond due to its ring.
                

                <h4>Alpha Helix</h4>
                The alpha helix involves the backbone carbonyl oxygen of residue i hydrogen bonding with the hydrogen attached to the nitrogen of the amide of residue i+4. 
                When viewed from above, the angle between the i'th residue, the center axis, and the i+1'th residue is approximately 100 degrees. 
                <br><br>
                Since all the hydrogen bonds in the helix share the same orientation, the alpha helix has an overall dipole, called the macro-dipole. 
                The overall dipole is positive at the N-terminal side and negative at the C-terminal side. 

                In cartoon depictions, alpah helices are shown as cylinders or a single helix. 
                <br><br>

                <h4>Beta sheets</h4>
                Beta sheets also involve backbone hydrogen bonding between amides, but the hydrogen bonded pairs are not as close in the sequence. 
                The basic unit of a beta sheet is the beta strand. In a beta strand, there is a 180 angle difference between consecutive residues. 
                If one were to imagine a plane on which the beta strand lied, the carbonyl oxygen would be alternatingly pointing up and down with each residue. A similarly geometry is present for the side chain positioning. 
                Thus, a beta strand does not hydrogen bond with itself. Another beta strand will be positioned above such that the plane for that strand is roughly parallel. Plane 1's upward facing nitrogens and carbonyl will hydrogen bond with plane 2's downward facing carbonyl and nitrogen respectively. 
                In reality, there is some curvature in the beta strand.
                More than two beta strands can be present as each plane still has another available side in this scenario. 
                <br><br>
                Another consideration for beta-sheets is whether the pairing is <b>parallel</b> or <b>anti-parallel</b>. 
                This refers to if the N-terminal ends of the strands are aligned or oppositely aligned in space. 
                Both are possible and there can be a mixture within a single beta sheet. 
                <br><br>
                In cartoon depictions, beta strands are arrows, where the arrow head points from n-terminus to the c-terminus.

                <h4>Collagen helix</h4>
                The collagen helix is another possible secondary structure which appears as a triple helix. 
                As the name suggests, it is present in types of collagen. 

                <h4>Motifs</h4>
                Combinations of beta strands/sheets and/or alpha helices are often observed in proteins 3D structures. 
                Common patterns are called <b>motifs</b>. 



                <h3>Tertiary Structure</h3>
                The tertiary structure involves further folding of the protein onto itself with various forms of interaction. 
                <ol>
                    <li>Covalent bonds (Cysteine-Cysteine disulfide bridge)</li>
                    <li>Ionic bonds</li>
                    <li>Polar bonds</li>
                    <li>Van der Waals force</li>
                </ol>


                <h3>Quartenary Structure</h3>
                Multiple tertiary structures can come together to form a quarternary structure

                <h2>Folding Models</h2>
                <h3>Anfinsen's Thermodynamic Hypothesis</h3>
                The thermodynamic hypothesis states that the native structure of a protein is determined just by its sequence. 
                It may be surprising that all the thermodynamic directions for complicated folding are encoded in the sequence. 
                This requires stability, uniqueness, and kinetic accessibility. 
                
                In general, this holds well, although notable counterexamples exist for each requirement.

                <h3>Levinthal's Paradox</h3>
                We saw that each residue had phi, psi, and omega angles that have energetically preferred values. And in a protein, there can easily be over a hundred residues, who can interact with each other. 
                Given how many parameters there are to optimize, how are proteins able to fold as quickly as they do? Anfinsen's thermodynamic hypothesis states that for a lot of proteins, the primary structure is sufficient to yield a unique 3D structure without outside assistance. 


                This is essentially Levinthal's paradox, that proteins are observed to be able to fold at a much faster speed than they would through random sampling of possible confirmations.
                <h3>Hierarchical Folding</h3>
                Hierarchical folding model attempts ot address part of Levinthal's paradox by stating that the search space is narrowed down significantly by proteins first folding into secondary structures and then tertiary structures. 
                A related concept is the <b>hydrophobic collapse process</b>.

                <h3>Difficult Problem in Optimization</h3>
                One of the biggest, if not the biggest, issues in optimization is local minima. 
                It is (relatively) simple to always go downhill to find the minimizer of a function, but what if there exists local minimizers in addition to the global minimizer. 

                <h4>Example</h4>
                In the below example, we have an optimization problem with two minima. 
                Say we are at the point x=-3, we can go downhill to the right and find a minimum around \(x\approx -2\). 
                However, this is not the global minimum. In order to then get to the global minimum, we would have to go uphill again to the right. 
                There is not real way for our algorithm to know that there exists a lower point somewhere else. 



                <img src="LocalMinima.png" alt="Optimization with Local Minima" style="margin:auto; display: block;  width:clamp(400px,50vw, 800px)"/>
                <button type="button" class="collapsible">Python Code</button>
                <div class="content">
                <code style="margin: auto; width: 50%;"><pre>
def energy_landscape(x):
    return -(x-3)*(x+2)-10*x**2-x**3+x**4+30

x_values = np.linspace(-5,5,100)
y_values = [energy_landscape(x) for x in x_values]
plt.figure()
plt.plot(x_values,y_values)
plt.xlabel("x")
plt.ylabel("f(x)")
plt.title("The Issue of Local Minima")
plt.xlim([-5,5])
plt.ylim([-20,200])
plt.show()   
</pre>
                </code></div>
                If this does not seem bad enough yet to you, consider all of the following:

                <ol>
                    <li>This is a function of only 1 variable and the value is quick to compute if we know the function</li>
                    <li>We have a graph, so it's obvious to us since we can visually see the whole function</li>
                    <li>We have no restrictions for the value of x</li>
                    <li>The function is continuous and smooth</li>
                </ol>

                <img src="Optimization2D.png" alt="Optimization 2D with Local Minima" style="margin:auto; display: block;  width:clamp(400px,50vw, 800px)"/>
                Python code:
                <code style="margin: auto; width: 50%;"><pre>
import numpy as np
import matplotlib.pyplot as plt

def rastrigin(x, y):
    """Compute the Rastrigin function."""
    return 20 + 0.2*x**2 + 0.1*y**2 - 2 * np.cos(0.5 * np.pi * x) - 1.5 * np.cos(0.5 * np.pi * y)

# Generate x and y data points
x = np.linspace(-5.12, 5.12, 400)
y = np.linspace(-5.12, 5.12, 400)
X, Y = np.meshgrid(x, y)
Z = rastrigin(X, Y)

# Plotting
fig = plt.figure(figsize=(14, 8))
ax = fig.add_subplot(111, projection='3d')
ax1 = ax.plot_surface(X, Y, Z, cmap='magma')
plt.colorbar(ax1)
ax.set_title("Rastrigin Function")
ax.set_xlabel('X axis')
ax.set_ylabel('Y axis')
ax.set_zlabel('Function value (Z)')
ax.view_init(45, -45)  # Adjust viewing angle for better visualization
plt.show()
                    
</pre>
                </code>
                
                Now let's imagine we are dealing with a function of 200 variables, which will take well over a thousand times longer for the computer to calculate the value of, we have constraints, and we cannot visualize the graph due to the high dimensionality. 
                The problem has gotten immeasurably harder.

                <h3>Folding Funnel</h3>
                The folding funnel hypothesis frames protein folding as an optimization problem for free energy with a search space that has a steep descent with the global minimum corresponding to the native state, similar to to a funnel shape. 
                The folding funnel permits some local minima in the landscape which correspond to misfolded states.

                <img src="Funnel.png" alt="Optimization with Local Minima" style="margin:auto; display: block;  width:clamp(400px,50vw, 800px)"/>
                <button type="button" class="collapsible">Python Code</button>
                <div class="content">
                <code style="margin: auto; width: 50%;"><pre>
def energy_landscape(x):
    return np.log(abs(x)+10)-0.05*np.sin(5*x)/x

x_values = np.linspace(-15,15,1000)
y_values = [energy_landscape(x) for x in x_values]
plt.figure()
plt.plot(x_values,y_values)
plt.xlabel("x")
plt.ylabel("f(x)")
plt.title("Sample Funnel")
plt.xlim([-10,10])
#plt.ylim([-20,200])
plt.show()
</pre>
                </code></div>

                <h2>Assistance in Folding</h2>
                <h3>Heat Shock Protein</h3>
                <h3>Chaperonins</h3>

                <h2>Misfolding</h2>
                <h3>Prions</h3>
                <h3>Alzheimer's</h3>

                <h2>Structure Determination</h2>

                <h2>Evolution</h2>
                <h3>Mutations</h3>
                <h3>BLOSOM</h3>
                BLOcks SUbstitution Matrices or BLOSOM are a way to compare sequence similarity between proteins, such as to see if they may evolutionarily related. 
                



                <h2>Applications</h2>
                <h3>Heat capacity</h3>
                Heat capacity is related to the number of ways in which something can utilize energy. For example, in a monoatomic gas, there are just 3 translational degrees of freedom, each contributing 
                \(\frac{1}{2}nR\)
                 to the heat capacity. In a diatomic molecule, you have the energy being taken up as vibrational and rotational energy, which add to the heat capacity.

                You always have certain degrees where you can input energy, such as translational and rotational degrees of freedom. Once you reach a certain temperature, thermal energy can now be taken up to disrupt interactions, such as H-bonding, increasing the heat capacity. 

                Once those molecular interactions have been broken, there are fewer ways that heat can be taken up for the protein, decreasing the heat capacity; you can't spend energy to break intermolecular forces that are already broken. Cp is still a bit higher than it was at lower temperature, however.
                <h4>Differential Scanning Calorimetry</h4>
                A popular experimental technique used to measure the heat capacity of proteins. This method measures the heat required to increase the protein sample's temperature compared to a reference, providing insights into heat capacity and thermal transitions.
                <p><a href="https://commons.wikimedia.org/wiki/File:CalibratingDSCcurve.png#/media/File:CalibratingDSCcurve.png"><img src="https://upload.wikimedia.org/wikipedia/commons/e/e5/CalibratingDSCcurve.png" alt="CalibratingDSCcurve.png" height="376" width="253"></a><br>By <a href="//commons.wikimedia.org/w/index.php?title=User:Schmin&amp;amp;action=edit&amp;amp;redlink=1" class="new" title="User:Schmin (page does not exist)">Schmin</a> - <span class="int-own-work" lang="en">Own work</span>, Public Domain, <a href="https://commons.wikimedia.org/w/index.php?curid=6792484">Link</a></p>
                
            </div>
            <div id="Exercises">
                <h2>Exercises</h2>
                <ol>
                    <li>How does the heat capacity of a protein relate to its denaturation?</li>
                    
                </ol>


            </div>
        </div>        <script>
            var coll = document.getElementsByClassName("collapsible");
            var i;
    
            for (i = 0; i < coll.length; i++) {
            coll[i].addEventListener("click", function() {
                this.classList.toggle("active");
                var content = this.nextElementSibling;
                if (content.style.display === "block") {
                content.style.display = "none";
                } else {
                content.style.display = "block";
                }
            });
            } 
            </script>
</body>
</html>